twitter/tweepy_scraper.py

```diff
@@ -41,6 +41,42 @@ def on_error(self, status_code):
             #returning False in on_data disconnects the stream
             return False
 
+
+def parse_status(status):
+    '''
+    Parse through the status and only keep the fields that are relevant to the model.
+    This will save hardware space.
+    :param status:
+    :return: a dictionary of all of the fields that we decided to keep
+    '''
```

Comment:
For docstrings, PEP8 style is to use three double quotes, instead of single quotes. https://www.python.org/dev/peps/pep-0008/#string-quotes